Unable to install a particular version

I tried to specify a particular version of Helm 3 like
```yaml
jobs:
  job:
    steps:
      - name: Install Helm 3
        uses: azure/setup-helm@v1
        with:
          version: 3.1.1
```
and got the following error:
```
##[error]Error: Failed to download Helm from location  https://get.helm.sh/helm-3.1.1-linux-amd64.zip
```

Using `latest` worked well.

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # Setup Helm
 #### Install a specific version of helm binary on the runner.
 
-Acceptable values are latest or any semantic version string like `1.15.0`. Use this action in workflow to define which version of helm will be used.
+Acceptable values are latest or any semantic version string like v2.16.7 Use this action in workflow to define which version of helm will be used.
 
 ```yaml
 - uses: azure/setup-helm@v1

--- a/lib/run.js
+++ b/lib/run.js
@@ -130,6 +130,9 @@ function run() {
         if (version.toLocaleLowerCase() === 'latest') {
             version = yield getStableHelmVersion();
         }
+        else if (!version.toLocaleLowerCase().startsWith('v')) {
+            version = 'v' + version;
+        }
         let cachedPath = yield downloadHelm(version);
         try {
             if (!process.env['PATH'].startsWith(path.dirname(cachedPath))) {

--- a/src/run.ts
+++ b/src/run.ts
@@ -120,6 +120,8 @@ async function run() {
     let version = core.getInput('version', { 'required': true });
     if (version.toLocaleLowerCase() === 'latest') {
         version = await getStableHelmVersion();
+    } else if (!version.toLocaleLowerCase().startsWith('v')) {
+        version = 'v' + version;
     }
 
     let cachedPath = await downloadHelm(version);

